### ISSUE TITLE
docs: improve the "Try it now" card in the landing page

### ISSUE THREAD

**Issue by bampakoa** [angular/angular#40179]
# 📚 Docs or angular.io bug report

### Description
I think that the **Try it now** card that exists on the bottom of the landing page could be further improved:

- To clearly indicate its importance. The white background does not really help on that.
- To better communicate the guidance for new users and try the framework. A button with a more intense color at the top of some pages, would be more approriate since users may not land in the first page by default.

### What's the affected URL?**
 https://angular.io

## 📷Screenshot

![image](https://user-images.githubusercontent.com/7170892/102538262-c0ed4c00-40b4-11eb-9634-fe473c7b0704.png)

## 🔥 Exception or Error
<pre><code>

</code></pre>

**Comment by Splaktar** [comment]
Yeah, that has weak clickability indicators for sure. A recognizable button would provide a better UX.

**Comment by petebacondarwin** [comment]
While we are at it. I think the on the resources page, the boxes containing each resource have weak clickability too. Perhaps we could solve that too?

**Comment by bampakoa** [comment]
If you need any help with ideas about the button, let me know. I have some good examples from other resources to share 😃

**Comment by Splaktar** [comment]
> I have some good examples from other resources to share 😃

Please do share.

**Comment by bampakoa** [comment]
Some of the landing pages that have a great UX, in my opinion, are:

- https://getbootstrap.com/
- https://ionicframework.com/
- https://scully.io/

If you notice, you will see that their Getting started guide is visible not only on the main page but also in the top bar of all pages, even if you scroll up and down. 

I took the opportunity and made some mockups below to explain better how I would imagine those changes in angular.io:

![image](https://user-images.githubusercontent.com/7170892/107189759-8cca4380-69f2-11eb-9749-3478a227e7da.png)

![image](https://user-images.githubusercontent.com/7170892/107190357-62c55100-69f3-11eb-8782-7da0db331f96.png)

![image](https://user-images.githubusercontent.com/7170892/107191439-d74cbf80-69f4-11eb-8cf5-5e8be8139fa5.png)

**In all the above examples, an icon along with the button would be great**

**Comment by bampakoa** [comment]
@aikidave is it possible to have your take on that?

**Comment by aikidave** [comment]
I'm afraid I am more of a technical writer than a UX designer, but I am all for anything that makes the site easier and more accessible!

**Comment by petebacondarwin** [comment]
@aikidave - please can you take another look at this. It is not really about the design but really about the structure of the navigation through these links.

**Comment by aikidave** [comment]
I like the direction that this is going. I wonder if we should have the "Try it Now" button on the landing page further up? "Why do you have to scroll all the way to the bottom to get to that link?" I wonder...

I also like the Try It Now button at the top of the page. 

@bampakoa Thanks for putting some thought into this! What would be the next steps if we move forward?
